### ISSUE TITLE
Warm reboot not working on latest SONiC images

### ISSUE THREAD

**Issue by BaluAlluru** [Azure/sonic-buildimage#4981]
**Description**
Error observed when executing "warm-reboot -v" command on SONIC Jenkins image 338

**Steps to reproduce the issue:**
1. Load the SONIC Jenkins image 338
2. Copied other SONIC image to box ( we copied SONIC Jenkins 350)
3. Execute the command 
    sudo sonic_installer install /home/admin/sonic-broadcom.bin
4. Execute "sudo warm-reboot -v". Observing Error ( Tried multiple times)
  
**Describe the results you received:**
root@sonic:/home/admin# sudo warm-reboot -v
Fri 17 Jul 2020 11:24:11 PM UTC Pausing orchagent ...
RESTARTCHECK failed
Fri 17 Jul 2020 11:24:24 PM UTC warm-reboot failure (10) cleanup ...
Fri 17 Jul 2020 11:24:24 PM UTC Cancel warm-reboot: code (1)

**Describe the results you expected:**
root@sonic:/home/admin# sudo warm-reboot -v
Fri Jul 17 13:59:50 UTC 2020 Pausing orchagent ...
Fri Jul 17 13:59:50 UTC 2020 Stopping nat ...
Fri Jul 17 13:59:51 UTC 2020 Stopped nat ...
Fri Jul 17 13:59:51 UTC 2020 Stopping radv ...
Fri Jul 17 13:59:51 UTC 2020 Stopping bgp ...
Fri Jul 17 13:59:51 UTC 2020 Stopped bgp ...
Fri Jul 17 13:59:53 UTC 2020 Initialize pre-shutdown ...
Fri Jul 17 13:59:54 UTC 2020 Requesting pre-shutdown ...
Fri Jul 17 13:59:54 UTC 2020 Waiting for pre-shutdown ...
Fri Jul 17 13:59:54 UTC 2020 Pre-shutdown succeeded ...
Fri Jul 17 13:59:54 UTC 2020 Backing up database ...
Fri Jul 17 13:59:55 UTC 2020 Stopping teamd ...
Fri Jul 17 13:59:55 UTC 2020 Stopped teamd ...
Fri Jul 17 13:59:55 UTC 2020 Stopping syncd ...
Fri Jul 17 14:00:11 UTC 2020 Stopped syncd ...
Fri Jul 17 14:00:11 UTC 2020 Stopping all remaining containers ...
Warning: Stopping telemetry.service, but it can still be activated by:
  telemetry.timer
Fri Jul 17 14:00:13 UTC 2020 Stopped all remaining containers ...
Fri Jul 17 14:00:15 UTC 2020 Rebooting with /sbin/kexec -e to SONiC-OS-master.350- ...
[  518.562641] kexec_core: Starting new kernel
...

**Additional information you deem important (e.g. issue happens only occasionally):**

    
root@sonic:/home/admin# show version
SONiC Software Version: SONiC.master.338-20698aa5
Distribution: Debian 10.4
Kernel: 4.19.0-6-2-amd64
Build commit: 20698aa5
Build date: Mon Jul  6 18:28:11 UTC 2020
Built by: johnar@jenkins-worker-4

    **Attach debug file `sudo generate_dump`:**
[sonic_dump_sonic_20200717_232503.tar.gz](https://github.com/Azure/sonic-buildimage/files/4938683/sonic_dump_sonic_20200717_232503.tar.gz)

**Comment by BaluAlluru** [comment]
opening this issue

**Comment by BaluAlluru** [comment]
In the 338 image we tested, xcvrd was not running due to wrong attribute names, which was later corrected in subsequent images.
I manually corrected the attribute names in this image to make xcvrd run.

Not seeing the below issue after xcvrd is running.

root@sonic:/home/admin# **sudo warm-reboot -v**
Fri 17 Jul 2020 11:24:11 PM UTC Pausing orchagent ...
RESTARTCHECK failed
Fri 17 Jul 2020 11:24:24 PM UTC warm-reboot failure (10) cleanup ...
Fri 17 Jul 2020 11:24:24 PM UTC Cancel warm-reboot: code (1)

Hence closing the issue.
